docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -428,7 +424,7 @@ To obtain a new JWT token once the current one is expired, repeat the above snip
 !!! example "Calling an API on behalf of example user"
 
     ```bash
-    curl --insecure https://localhost:8443/core-data/api/v2/ping -H "Authorization: Bearer $(cat user-jwt.txt)"
```

Comment:
To ensure consistency, the file name 'user-jwt.txt' should be replaced with 'id_token.txt'. This replacement should also be made in any corresponding references to the file, including line 449.